api/src/main/java/org/pmiops/workbench/actionaudit/bucket/BucketAuditEntry.java

```diff
@@ -0,0 +1,69 @@
+package org.pmiops.workbench.actionaudit.bucket;
```

Comment:
There are few other options of Pojo class in Java. 

- Some code in Workbench defines that in workbench-api.yaml then generated via swagger codeGen
- McTerra uses autovalue ([go/autovalue](go/autovalue))
Personally I prefer autovalue and it would work better than swagger in your case where you need Time type...
Happy to chat offline if you have preference on this. 

Some other reference: https://docs.google.com/document/d/1ugU-O1s2mh7wfq63kANdhxgU9ph-J6VsCkYjiJv3NlA/edit